style.css

```diff
@@ -167,15 +209,31 @@ body {
   font-size: 24px;
 }
 
-.artist-img {
-  width: auto;
-  height: 100px;
+.artists-main-box {
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+  margin-bottom: 50px;
+  padding: 30px 0 0 40px;
 }
 
 .artist-box {
   display: flex;
-  flex-direction: row;
-  margin: 40px 0 40px 40px;
+  padding: 19px 0 0 19px;
+  background-image: url(./imgs-capstone/chess-table.jpg);
+  background-repeat: no-repeat;
+  background-size: 50px;
+}
+
+.future,
+.lil-uzi,
+.kodak-black,
+.lil-wayne {
+  display: none;
+}
+
+.artist-img {
```

Comment:
- [ ] As mentioned by the previous reviewer, your nav menu 'style is a bit different from the original one, you should make some changes to match the design as she showed you and request for a next review, the other things were fixed correctly, congratulations

YOUR DESIGN

<img width="437" alt="1" src="https://user-images.githubusercontent.com/77843050/233491258-80c4fe7c-b5ef-484e-9dfd-a482f6555d39.png">


figma

<img width="386" alt="1" src="https://user-images.githubusercontent.com/77843050/233491401-1c832d9d-8168-42e2-a4aa-28ada14b1945.png">